defaults/test_default.yaml

```diff
@@ -242,3 +242,11 @@ validate_large_collections: false
 scylla_d_overrides_files: []
 
 run_commit_log_check_thread: true
+
+# teardown validators
+teardown_validators:
+  scrub:
```

Comment:
I do agree documenting for those should be in it's own markdown, explaining the why and giving examples of usages.